tests/test_objects.py

```diff
@@ -40,14 +41,15 @@
 }
 
 DEFAULT_MX_RECORD = {
-    '_ref': 'record:mx/ZG5zLmJpbmRfbXgkLjQuY29tLm15X3pvbmUuZGVtby5teC5kZW1vLm15X3pvbmUuY29tLjE'
-            'mx.demo.my_zone.com/my_dns_view',
+    '_ref': 'record:mx/%s'
+            'mx.demo.my_zone.com/my_dns_view' % REC,
     'view': 'my_dns_view',
     'name': 'mx.demo.my_zone.com',
     'preference': '1',
     'mail_exchanger': 'demo.my_zone.com'
 }
 
+
```

Comment:
extra line

Reply:
Pyflake mandates that 2 blank lines be used before a class declaration, so I think it makes sense here :)